contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -720,7 +976,7 @@ void AlembicScene::readTags( NameList &tags, int filter ) const
 
 void AlembicScene::writeTags( const NameList &tags )
 {
-	throw IECore::NotImplementedException( "AlembicScene::writeTags" );
+	IECore::msg( IECore::Msg::Warning, "AlembicScene::writeAttribute", "Not implemented" );
```

Comment:
So Maya Sets don't export to Alembic at all?

Ok, we can push ahead for now to get a version out, but I don't think we'll be able to do much format comparison until we're at parity with the features (ie. it'd be hard to compare 2 gaffer renders at IE without supporting sets and attributes in both formats).